app/models/vm_reconfigure_request.rb

```diff
@@ -79,6 +79,17 @@ def self.validate_request(options)
     errors
   end
 
+  def self.make_request(request, values, requester, auto_approve = false)
+    values[:request_type] = :vm_reconfigure
+
+    requests = ApplicationRecord.group_ids_by_region(values[:src_ids]).collect do |_region, ids|
+      super(request, values.merge(:src_ids => ids), requester, auto_approve)
+    end
+
+    return requests.first if requests.count == 1
```

Comment:
I think we need to move toward returning an array of requests (ActiveRecord or Api).  Since the caller doesn't care in this case, I'd like this method to return an array only